paymentsheet/src/main/java/com/stripe/android/link/ui/PrimaryButton.kt

```diff
@@ -52,7 +51,7 @@ internal fun PrimaryButton(
         LocalContentAlpha provides
             if (state == PrimaryButtonState.Disabled) ContentAlpha.disabled else ContentAlpha.high
     ) {
-        Box(modifier.padding(vertical = 16.dp)) {
+        Box(modifier) {
```

Comment:
👀 Better to allow the caller to add margins, which I had to do at all the call sites in this PR